src/renderer/hooks/useNodeMenu.tsx

```diff
@@ -64,12 +75,90 @@ export const useNodeMenu = (
             <MenuDivider />
             <MenuItem
                 icon={<CloseIcon />}
-                onClick={() => {
-                    clearNodes([id]);
+                ref={resetMenuParentRef}
+                onClick={(e) => {
+                    e.stopPropagation();
+                    e.preventDefault();
+                    setShowResetSubMenu(true);
+                }}
+                onContextMenu={() => {
+                    setShowResetSubMenu(false);
+                }}
+                onMouseEnter={() => {
+                    setShowResetSubMenu(true);
+                }}
+                onMouseLeave={() => {
+                    setShowResetSubMenu(false);
+                }}
+                onMouseOver={() => {
+                    setShowResetSubMenu(true);
```

Comment:
https://www.youtube.com/watch?v=Xn4yAbA9cZw

Please use CSS. A simple `.container:not(:hover) > .child { display: none; }` could go so far.

Reply:
This either doesn't work the way you think it does or I'm doing something wrong

Reply:
Ok, i had to put the child classname on a div wrapping the submenu list... wtf